thunder/common.py

```diff
@@ -629,6 +630,7 @@ def wait_for_future(f: FutureTensorProxy) -> TensorProxy:
 # TODO Consider making this faster by reusing more data
 # TODO Create a general mechanism for running traces that produces reproducible provenance and the
 #   appropriate error checks
+@thunder.core.profile.profile("transform_for_execution")
```

Comment:
Happy to remove any/all of these for now if that makes your refactoring easier, @IvanYashchuk. Just wanted a couple places so that the decorator would actually get tested in CI.

Reply:
Another way to test the decorator in CI is to add a dedicated test.

Reply:
Cool, done.